Bookish/Bookish.DataAccess/BarcodeService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.IO;
+using System.Linq;
+using System.Text;
+using BarcodeLib;
+
+namespace Bookish.DataAccess
+{
+    public interface IBarcodeService
+    {
+        IEnumerable<int> GetBarcodeIds(string isbn, int copies, string imageFolder);
+    }
+    
+    public class BarcodeService : IBarcodeService
+    {
+        private readonly ILibraryService libraryService;
+        public BarcodeService(ILibraryService libraryService)
+        {
+            this.libraryService = libraryService;
+        }
+
+        public IEnumerable<int> GetBarcodeIds(string isbn, int copies, string imageFolder)
+        {
+            try
+            {
+                Clean(imageFolder);
+            }
+            catch
+            {
+                // ignored
```

Comment:
You could at least log the exception here 🙂

Reply:
How do I use the logger within the barcode service?

Reply:
Yes, it should be by dependency injection 🙂 Are you using a logger in this project so far? If you are you should be able to just inject it here. If you don't then feel free to just use console log for now. You can always setup a proper logger in a different PR. I'd recommend [serilog](https://serilog.net/) as that's a popular one and it's what we use on HEE 🙂

Reply:
Currently the only logger (i haven't actually used it) is the one that came when the project was created which is `Microsoft.Extension.Logging` which has a `ILogger<HomeController>` in the home controller. I can look into using serilog instead and just ignoring this one

Reply:
Oh yes, I see you're using that in HomeController. That's is the correct type and the correct way to inject it 🙂 If you were using serilog that wouldn't affect how you use it, just how it's set up in Program.cs (see [this example](https://github.com/TechnologyEnhancedLearning/DLSV2/blob/master/DigitalLearningSolutions.Web/Program.cs)). 

It doesn't look like you have any setup at the moment, so I think you're just using the default logger which I'd guess just logs to the console although I haven't tried using it. Fine to keep using that if it works fine. If you want to try setting up some permanent logs you can add serilog as in that example and make it log to a file like this:
```
Log.Logger = new LoggerConfiguration()
    .WriteTo.Console()
    .WriteTo.File("log-.txt", rollingInterval: RollingInterval.Day)
    .CreateLogger();
```
That would log both to a console and to a file, more info [here](https://github.com/serilog/serilog/wiki/Configuration-Basics) 🙂